### ISSUE TITLE
False positive: use_g_autoptr_inline_cleanup

### ISSUE THREAD

**Issue by blandford** [bilelmoussaoui/gobject-lint#37]
Minor problem but it should suggest g_autofree() instead of g_autoptr().

`theme-filter-test.c:65:5: error: Consider using g_autoptr(TestContext) ctx to avoid manual cleanup [use_g_autoptr_inline_cleanup]`

Relevant code:

```C
typedef struct {
    GMainLoop *loop;
    GListModel *model;
    guint index;
    GtkStringList *theme_words;
    IpuzCharset *theme_lengths;
    TaskRunner *runner;
    guint n_matches;
    guint n_misses;
} TestContext;

static void
test_theme_filtering (void)
{
    TestContext *ctx = g_new0 (TestContext, 1);
...
    g_free (ctx);
```
